contracts/Proxy/DelegateManagable.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.4.18;
+
+import './BaseProxy.sol';
+import './DelegateManagerInterface.sol';
+
+contract DelegateManagable is BaseProxy {
```

Comment:
thinking about having everything that extends Proxy be named with "Proxy" .. could call this `ManagedProxy` or something like that